src/plugins/ImageSearch/index.tsx

```diff
@@ -16,14 +17,14 @@ function handler({ dispatch, viewRegistry, origin, id }: ActionHandlerI): boolea
   return true
 }
 
-export function ImageSearchPlugin(): any {
+export const ImageSearchPlugin: Plugin.InitFunction = () => {
   const { state, dispatch } = useNavigation()
   const { viewId: origin } = useView()
   const viewRegistry = state.viewRegistry
   const query = useQuery()
   const { id } = query
 
-  return ({
+  return {
     class: 'block',
     name: 'tt/visual/search',
     componentEntry: {
```

Comment:
Unsure what the proper type is here:
It says we're missing `component` in `componentEntry`. Not sure if we're missing something.

Reply:
the `component` is the acting parent/wrapper, if I understand it correctly. In this case, there isn't a parent component.

Reply:
You can either define the function as a Plugin.IniFunction. Or define it to return a Plugin.Definition. Either way is fine. But it does not matter for the issue at hand.

A componentEntry must always have a component. But this plugin does not need a componentEntry. So remove that entirely.